src/modules/Dropdown/Dropdown.js

```diff
@@ -687,6 +693,8 @@ export default class Dropdown extends Component {
   // ----------------------------------------
 
   renderText = () => {
+    if (this.props.trigger) return this.props.trigger
```

Comment:
Agreed. Also, many of the other renders (renderLabels, etc) will not be used when there's a `trigger` present since those are all `selection` related. I think it's probably OK for now but it would probably be good to think about how to split these two separate use-cases out.